dids/src/main/kotlin/web5/sdk/dids/methods/dht/DidDhtDocumentMetadata.kt

```diff
@@ -0,0 +1,13 @@
+package web5.sdk.dids.methods.dht
+
+import web5.sdk.dids.didcore.DidDocumentMetadata
+
+/**
+ * Did document metadata for did:dht that extends the base did document metadata.
```

Comment:
again, links to spec?

Reply:
hmm, i can't seem to find where `type` is defined in did dht document spec - @decentralgabe ? best i can find is https://did-dht.com/#type-indexing but that's about how to use types, not where types are defined in the did dht doc metadata section